activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -909,17 +909,16 @@ def arel(aliases = nil) # :nodoc:
       @arel ||= build_arel(aliases)
     end
 
-    protected
-      # Returns a relation value with a given name
-      def get_value(name) # :nodoc:
-        @values.fetch(name, DEFAULT_VALUES[name])
-      end
+    # Returns a relation value with a given name
+    def get_value(name) # :nodoc:
```

Comment:
#32709 is caused f0e6ecc9966f5a569313658bedff54bde8517e65 due to `delegate_methods` won't contain `:get_value` since `get_value` has not been `public_instance_methods`.
`@values` in `CollectionProxy` is always empty, so `get_value(name)` should be delegated as `scope.get_value(name)`.

https://github.com/rails/rails/blob/03721d9379c0eeb8b0c93c92301ba7a9caa436a2/activerecord/lib/active_record/associations/collection_proxy.rb#L1123-L1130